freeipa/src/main/resources/freeipa-salt/salt/freeipa/init.sls

```diff
@@ -58,4 +58,10 @@ net.ipv6.conf.lo.disable_ipv6:
     - user: root
     - group: root
     - mode: 700
-    - source: salt://freeipa/scripts/repair.sh
\ No newline at end of file
+    - source: salt://freeipa/scripts/repair.sh
+
+disable_old_tls_for_ldap_server:
+  file.append:
+    - name: /usr/share/ipa/updates/20-sslciphers.update
+    - text: 'only:sslVersionMin: TLS1.2'
+    - unless: grep "sslVersionMin" /usr/share/ipa/updates/20-sslciphers.update
```

Comment:
do we need the unless?
`The text will not be appended if it already exists in the file. A single string of text or a list of strings may be appended.`
https://docs.saltproject.io/en/latest/ref/states/all/salt.states.file.html#salt.states.file.append

Reply:
Correct. Fixed it.